### DISCUSSION TITLE
Duda Lab 05.02 task 1

### DISCUSSION THREAD

**Discussion by manuelagutierrezc** [rramosp/ai4eng.v1#177] [05 - GESTIÓN DE MODELOS]
Hola. Tengo una pregunta sobre la tarea 1, he hecho una solución, pero no sé qué está incorrecto. Agradezco su ayuda.

https://colab.research.google.com/drive/1waBM0zobJkxwO3UtNi8jr-SW6-2aYVYq?usp=sharing

**Comment by rramosp** [comment]
manuela, no podemos ver tu notebook ... pega un pantallazo de tu solución
acà

On Thu, Jul 28, 2022 at 10:19 AM Manuela Gutiérrez ***@***.***>
wrote:

> Hola. Tengo una pregunta sobre la tarea 1, he hecho una solución, pero no
> sé qué está incorrecto. Agradezco su ayuda.
>
>
> https://colab.research.google.com/drive/1waBM0zobJkxwO3UtNi8jr-SW6-2aYVYq?usp=sharing
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/177>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIJLK2V335DPPA4APDLVWKQHTANCNFSM545VQVQQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.

**Comment by manuelagutierrezc** [comment]
`def split_data(X, y, pct):
    
    assert len(X)==len(y), "X and y must have the same length"
    assert pct>0 and pct<1, "pct must be in the (0,1) iterval"
    
    n = len(y)
    permutation = np.random.permutation(n-1)
    n1_elements = int(n*pct)
    permutation_partition_1 = permutation[:n1_elements]
    permutation_partition_2 = permutation[n1_elements:]
    X1 = X[permutation_partition_1,:]
    X2 = X[permutation_partition_2,:]
    y1 = y[permutation_partition_1,]
    y2 = y[permutation_partition_2,]
    return X1, X2, y1, y2`
Es la tarea 1.

**Comment by Felipe-RA** [comment]
Saludos Manuela

Revisa el intervalo en el que estás generando las permutaciones!
